.github/workflows/roll_torch_mlir.yml

```
@@ -0,0 +1,60 @@
+name: Roll PyTorch
+
+on:
+  schedule:
+    - cron: '0 13 * * *'
+  workflow_dispatch:
+
+jobs:
+  build_linux:
+    name: roll torch-mlir
+    needs: setup
+    runs-on:
+      - self-hosted
+      - runner-group=${{ needs.setup.outputs.runner-group }}
+      - environment=${{ needs.setup.outputs.runner-env }}
+      - cpu
+      - os-family=Linux
```

As we don't need lots of computation power here, maybe small Github hosted runner will be enough
```suggestion
    runs-on: ubuntu-20.04-64core
```